DocGen4/Output/References.lean

```diff
@@ -0,0 +1,77 @@
+import DocGen4.Output.Template
+
+/-!
+
+# Generic functions for references support
+
+This file contains functions for references support,
+independent of actual implementation.
+
+The main function is `DocGen4.preprocessBibFile` which preprocess
+the contents of bib file using user provided `process` function,
+and save the bib file and processed json file to output directory.
+
+Currently we have an implementation of `process` function using `pybtex`,
```

Comment:
I think this doc-string paragraph is not necessary here. The implementation is generic and we will probably forget to update it as the implementation changes.